src/restconfig/src/test/java/org/geoserver/rest/catalog/LayerControllerTest.java

```diff
@@ -268,4 +269,36 @@ public void testPutWorkspaceAlternateStyle() throws Exception {
         assertXpathEvaluatesTo("http://localhost:8080/geoserver"+ROOT_PATH+"/workspaces/cite/styles/foo.xml", 
             "//styles/style/atom:link/@href", dom );
     }
+
+    @Test
+    public void testPutDefaultWMSInterpolationMethod() throws Exception {
+        Catalog cat = getCatalog();
+        LayerInfo l = cat.getLayerByName("cite:Buildings");
+        assertNotNull(l);
+        assertNull(l.getDefaultWMSInterpolationMethod());
+
+        Document dom = getAsDOM(ROOT_PATH + "/layers/cite:Buildings.xml",200);
+        assertEquals("layer", dom.getDocumentElement().getNodeName());
+        assertXpathNotExists("/layer/defaultWMSInterpolationMethod", dom);
+
+        String xml =
+            "<layer>" +
+                "<defaultWMSInterpolationMethod>" +
+                    "Nearest" +
+                "</defaultWMSInterpolationMethod>" +
+                "<enabled>true</enabled>" +
+            "</layer>";
+        MockHttpServletResponse response =
+            putAsServletResponse(ROOT_PATH + "/layers/cite:Buildings", xml, "application/xml");
+        assertEquals(200, response.getStatus());
+
+        l = cat.getLayerByName("cite:Buildings");
+        assertNotNull(l.getDefaultWMSInterpolationMethod());
+        assertEquals(LayerInfo.WMSInterpolation.Nearest, l.getDefaultWMSInterpolationMethod());
+
+        dom = getAsDOM(ROOT_PATH + "/layers/cite:Buildings.xml",200);
+        print(dom);
```

Comment:
Ha! As I was writing that line, I thought to myself: "You are going to forget to remove this"; and of course, that's exactly what happened :wink: